pkg/scale/uint128.go

```diff
@@ -161,3 +161,8 @@ func (u *Uint128) UnmarshalJSON(data []byte) error {
 	u.Lower = dec.Lower
 	return nil
 }
+
+// MarshalJSON converts Uint128 to []byte.
+func (u *Uint128) MarshalJSON() ([]byte, error) {
```

Comment:
Should we add an Unmarshal method as well?

Reply:
Unmarshal method is already there.